### DISCUSSION TITLE
Remove tree and wave emoji from lualine

### DISCUSSION THREAD

**Discussion by alichtman** [LazyVim/LazyVim#2504] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/20600565/97b6ccc7-375d-42ea-a76e-a18ead6330d7)

I have these icons in my lualine, and I can't figure out what they mean / where they're coming from.

Does anyone recognize them? My [config is here](https://github.com/alichtman/dotfiles/tree/main/.config/nvim-lazyvim).

**Comment by dpetka2001** [comment, marked as answer]
https://github.com/alichtman/dotfiles/blob/9b58057c6d26ba46c300c973613c7a814e110f2e/.config/nvim-lazyvim/lua/util/dashboard/config.lua#L15-L20

**Reply by alichtman** [reply]
Thanks. I should have found that. I'll post an update when I figure out what to do about it.
